embc-app/ClientApp/src/app/shared/components/date-time-picker/date-time-picker.component.scss

```diff
@@ -0,0 +1,9 @@
+@import "~src/styles/_defaults.scss";
```

Comment:
Should this be _defaults.scss or defaults.scss?
If the latter then this should be fixed similarly in all components.

Reply:
sure, I'll remove it